packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -893,14 +893,15 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
    * Updates the agent device type.
    * This method allows the agent to change their device type (e.g., from BROWSER to EXTENSION or anything else).
    * It will also throw an error if the new device type is the same as the current one.
-   * @param data type is AgentDeviceUpdate - The data required to update the agent device type, including the new login option and dial number.
+   * @param data type is AgentDeviceUpdate - The data required to update the agent device type, including the new login option, new team and dial number.
```

Comment:
We can remove the "new"
```suggestion
   * @param data type is AgentDeviceUpdate - The data required to update the agent device type, including the new login option, team and dial number.
```

Reply:
Should we also consider changing the name of this method? `updateDeviceType` doesn't seem appropriate anymore

Reply:
Fair - I think `AgentProfileUpdate` would work best in this case.